subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -1,5 +1,6 @@
 /*
```

Comment:
Could you place the shell changes to a separate commit

Reply:
updated as above requests